src/bitmessagemain.py

```diff
@@ -73,17 +77,19 @@ def connectToStream(streamNumber):
     selfInitiatedConnections[streamNumber] = {}
 
     if isOurOperatingSystemLimitedToHavingVeryFewHalfOpenConnections():
-        # Some XP and Vista systems can only have 10 outgoing connections at a time.
+        # Some XP and Vista systems can only have 10 outgoing connections
+        # at a time.
         state.maximumNumberOfHalfOpenConnections = 9
     else:
         state.maximumNumberOfHalfOpenConnections = 64
     try:
         # don't overload Tor
-        if BMConfigParser().get('bitmessagesettings', 'socksproxytype') != 'none':
+        if BMConfigParser().get(
+                'bitmessagesettings', 'socksproxytype') != 'none':
             state.maximumNumberOfHalfOpenConnections = 4
     except:
         pass
```

Comment:
Write log in the place of pass

Reply:
Maybe it's better to remove try .. except?
```python
if BMConfigParser().safeGet('bitmessagesettings', 'socksproxytype') != 'none':
    state.maximumNumberOfHalfOpenConnections = 4
```

Reply:
safeGet shouldn't throw an exception, if it does, it should be fixed. I agree to have the try/except removed, and perhaps add the socksproxytype none to the config defaults.